patterns/pages/faculty_staff_spotlight_page.php

```diff
@@ -1,97 +1,93 @@
+
+          <!-- wp:post-excerpt /-->
+          <!-- /wp:post-template -->
+        </div>
+        <!-- /wp:query -->
 
-                <!-- wp:group {"style":{"color":{"background":"#0055b7"}},"className":"spotlight-submission-footer"} -->
-                <div
-                    class="wp-block-group spotlight-submission-footer has-white-color has-text-color has-background"
-                    style="background-color: #0055b7"
-                >
-                    <!-- wp:heading -->
-                    <h2>Spotlight Submissions</h2>
-                    <!-- /wp:heading -->
+        <!-- wp:group {"style":{"color":{"background":"#0055b7"}},"className":"spotlight-submission-footer has-white-color has-text-color"} -->
+        <div
+          class="wp-block-group spotlight-submission-footer has-white-color has-text-color has-background"
+          style="background-color: #0055b7"
+        >
+          <!-- wp:heading -->
+          <h2>Spotlight Submissions</h2>
+          <!-- /wp:heading -->
 
-                    <!-- wp:paragraph -->
-                    <p>
-                        If you are interested in being profiled as part of the
-                        spotlight series or would like to nominate a colleague,
-                        email
-                        <a
-                            href="mailto:communications.med@ubc.ca"
-                            target="_blank"
-                            rel="noreferrer noopener"
-                            >communications.med@ubc.ca</a
-                        >.
-                    </p>
-                    <!-- /wp:paragraph -->
+          <!-- wp:paragraph -->
+          <p>
+            If you are interested in being profiled as part of the spotlight
+            series or would like to nominate a colleague, email
+            <a
+              href="mailto:communications.med@ubc.ca"
+              target="_blank"
+              rel="noreferrer noopener"
+              >communications.med@ubc.ca</a
+            >.
+          </p>
+          <!-- /wp:paragraph -->
 
-                    <!-- wp:buttons {"layout":{"type":"flex","justifyContent":"center"}} -->
-                    <div class="wp-block-buttons">
-                        <!-- wp:button {"style":{"color":{"background":"#002145"}},"className":"is-style-blue-btn"} -->
-                        <div class="wp-block-button is-style-blue-btn">
-                            <a
-                                class="wp-block-button__link has-background"
-                                style="background-color: #002145"
-                                >Contact Us</a
-                            >
-                        </div>
-                        <!-- /wp:button -->
-                    </div>
-                    <!-- /wp:buttons -->
-                </div>
-                <!-- /wp:group -->
+          <!-- wp:buttons {"layout":{"type":"flex","justifyContent":"center"}} -->
```

Comment:
Remove all styling from buttons. Leave to default, default styling fixed in another commit.